packages/twenty-server/src/workspace/messaging/services/create-contact.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+
+import { EntityManager } from 'typeorm';
+import { v4 } from 'uuid';
+
+import { DataSourceEntity } from 'src/metadata/data-source/data-source.entity';
+import { capitalize } from 'src/utils/capitalize';
+@Injectable()
+export class CreateContactService {
+  constructor() {}
+
+  async createContactFromHandleAndDisplayName(
```

Comment:
```suggestion
export type ContactToCreate {
   handle: string;
   displayName: string;
}

  async createContacts(
     contactsToCreate: ContactToCreate[]
```

Reply:
Could we make this work this way and change saveMessageParticipants so that it works as a batch? I feel like we could improve a bit this area